File: 3k/1s/cpp/1. Conversation/Formalist.cpp
class Formalist: public Person
{
protected:
    char* getTypeText()
    {
        return "Формалист";
    }

public:
    Formalist(char* newName, int newAge)
    {
        init(newName, newAge);
    }

    const void Hello(Person* &personTwo) {
        cout << "Здравствуйте, " << personTwo->getName();
    }
};

File: 3k/2s/java/5. Conversation/src/com/company/Formalist.java
package com.company;

public class Formalist extends Person
{
    public Formalist(String newName, int newAge, int newSex)
    {
        super(newName, newAge, newSex);
    }

    public String getTypeText()
    {
        return "Формалист" + getTypeTextSuffix();
    }
    
    public void hello(Person twoPerson)
    {
        System.out.print("Здравствуйте, " + twoPerson.getName());
    }
}
